.rubocop_todo.yml

```diff
@@ -114,3 +114,9 @@ Style/SlicingWithRange:
 # SupportedStyles: use_perl_names, use_english_names
 Style/SpecialGlobalVars:
   EnforcedStyle: use_perl_names
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Style/WordArray:
+  Exclude:
+    - 'app/models/expression.rb'
```

Comment:
I'm not 100% sure this is the best way to disable rubocop on the one line, open to any ideas

Reply:
`# rubocop:disable SomeRule` if you're doing it inline

Reply:
it is done in the file which has the offending code, not the rubocop config

Reply:
oh i see, that is much simpler, will change that now